src/utils/manager.py

```diff
@@ -128,10 +145,9 @@ def fetch(target: str) -> Optional[MountInfo]:
     Returns:
         Optional[MountInfo]: Mount information. None if NFS share is not mounted.
     """
-    # Check if target is NFS URL. If so, convert to `mount.nfs` endpoint format
-    # since that is what is stored in /proc/mounts.
-    if _nfs_url_check.match(target):
-        target, port = _translate(target)
+    # Convert to `mount.nfs` endpoint format, since that is what is stored in /proc/mounts.
```

Comment:
[Nit]: Convert and translate are synonyms, but let's use `Translate` rather than `Convert` just to keep things uniform.

 ```suggestion
    # Translate to `mount.nfs` endpoint format. This format is what is stored in /proc/mounts.
```